doc/source/tutorial/stats.rst

```diff
@@ -200,6 +201,9 @@ seed an internal ``Generator`` object:
 For further info, see `NumPy's documentation
 <https://numpy.org/doc/stable/reference/random/index.html>`__.
 
+To learn more about the random number samplers implemented in SciPy, see
+the :ref:`sampling tutorial <non-uniform-random-number-sampling>`
```

Comment:
OK. Just to be sure, you are asking for something like this, right?

```suggestion
To learn more about the random number samplers implemented in SciPy, see
:ref:`non-uniform random number sampling tutorial
<non-uniform-random-number-sampling>` and `quasi monte carlo tutorial
<stats>`

If there is a tutorial for uniform quasi-random number samplers, I can link that here.
```

Reply:
This would be better to me, yes.

So we just have these tutorials and the QMC one for sampling. On top of my head, we (I or Matt might do it soon) should do one on `NumericalInverseHermite` and also link this to what we can do with the `ppf` and `rvs` of the distributions. For now it's fine, but we should think about making a story around this topic.